recipe/meta.yaml

```
@@ -10,35 +10,46 @@ source:
   sha256: f188e864cc0fed9fb141aff73c21bfaab783a1dd1b97ab1eff44340aed68f50b
 
 build:
-  noarch: python
   script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
   number: 0
+  skip: True  # [py<39]
 
 requirements:
   host:
-    - python {{ python_min }}
+    - python
     - pip
     - setuptools
+    - wheel
   run:
-    - python >={{ python_min }}
+    - python
     - requests >=2.7,<3.0
     - backoff >=2.1,<3.0
     - python-dateutil >=2.2,<3.0
     - pyjwt >=2.10.1,<2.11.0
 
 test:
+  source_files:
+    - segment/analytics/test
   imports:
     - segment.analytics
   commands:
     - pip check
+    - pytest segment/analytics/test -v -k "not (test_upload_interval or test_oauth)"  # Skip flaky tests
   requires:
-    - python {{ python_min }}
+    - python
     - pip
+    - pytest
+    - mock
+    - pylint
+    - flake8
+    - cryptography
 
 about:
   home: https://github.com/segmentio/analytics-python
   summary: The hassle-free way to integrate analytics into any python application.
+  description: The hassle-free way to integrate analytics into any python application.
```

Slightly better description that was in their setup.py


```suggestion
  description: |
      Segment is the simplest way to integrate analytics into your application.
      One API allows you to turn on any other analytics service. No more learning
      new APIs, repeated code, and wasted development time.
```